ultimate_cron.module

```diff
@@ -805,75 +418,34 @@ function ultimate_cron_get_hooks($reset = FALSE) {
 
   $hooks = array();
   // Generate list of jobs provided by modules.
-  $modules = module_list();
+  $modules = array_keys(Drupal::moduleHandler()->getModuleList());
   foreach ($modules as $module) {
     $hooks += ultimate_cron_get_module_hooks($module);
   }
 
   // Generate list of jobs provided by plugins.
-  $hooks += ultimate_cron_get_plugin_hooks();
-
+  // @todo: Port to Drupal 8
+  // $hooks += ultimate_cron_get_plugin_hooks();
```

Comment:
Hm, looks like a duplicate of the other code. Drop as well then, also that fucntion.